assets/samples/dynakube/v1beta3/logMonitoring.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: dynatrace.com/v1beta3
+kind: DynaKube
+metadata:
+  name: dynakube
+  namespace: dynatrace
+spec:
+  apiUrl: https://ENVIRONMENTID.live.dynatrace.com/api
+
+  # Link to api reference for further information: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters#tabgroup--apiversions--v1beta2
+
+  # This feature is still experimental, for more context please refer to: https://docs.dynatrace.com/docs/ingest-from/setup-on-k8s/reference/dynakube-parameters
```

Comment:
We will not be calling it "experimental". I still need to clarify if we should have the notice here or if it's enough to have this information in the official documentation

Reply:
Then I will removed this part. `This feature is still experimental, `

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4160/commits/232423ff5382b8502a544ea25f32d512afd24897